settings.gradle.kts

```diff
@@ -8,6 +8,22 @@ pluginManagement {
   }
 }
 
+dependencyResolutionManagement {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+    maven("https://hypertrace.jfrog.io/artifactory/maven")
+  }
+  versionCatalogs {
+    create("commonLibs") {
+      from("org.hypertrace.bom:hypertrace-version-catalog:+")
```

Comment:
so this would get the latest catalog and bom whenever locks are refreshed.

I was thinking we'd add a GHA job - probably scheduled with a manual option - that regenerates locks and raises a new PR. That means a bom change would percolate to all consuming repos on their own schedule, or can be manually forced. All builds continue to be reproducible due to lock.

Reply:
Which project is publishing this catalog? I checked here https://github.com/hypertrace/hypertrace-bom, did not find one.